daemon/mgr/container.go

```diff
@@ -414,6 +410,23 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 	}, nil
 }
 
+// set manager log config to the container's log config if the container's log config is not specified.
+func (mgr *ContainerManager) setDefaultLogConfig(logConfig *types.LogConfig) {
+	if logConfig == nil {
```

Comment:
I don't think we can set the value to the config, since it is not pointer to pointer